simpleclient_servlet/src/main/java/io/prometheus/client/exporter/MetricsServlet.java

```diff
@@ -14,6 +14,26 @@
 import java.util.HashSet;
 import java.util.Set;
 
+/**
+ * The MetricsServlet class exists to provide a simple way of exposing the metrics values.
+ *
+ * <p>This servlet might be used with a filter such the {@link io.prometheus.client.filter.MetricsFilter}
+ *
+ * <p>Usage example (in webapp's {@code web.xml}):
```

Comment:
What's a web.xml?

This seems specific to one particular framework, and only one way that this could be used. The changes to this file don't belong here.

Reply:
The web.xml file is the deployment descriptor for a Servlet-based Java web application. It's pretty common.
The XML snippet that is in MetricsFilter.java is also referring to that web.xml.

A user of the servlet may want to know how to use it, this way is the most straightforward.

Reply:
I personally have never heard of it, nor have I ever needed to use it to use servlets. This sounds like something specific to the framework you are using, and not generally applicable and thus doesn't belong here.

Reply:
I presumed it was something Spring related, as it came in in the same PR. Claiming that this applies to all web applications seems a bit broad.

Reply:
This PR is "Spring-free" ...

I don't claim it applies to all web application. But for sure, its the most common way of deploying a very simple servlet (especially when not using an extra framework). It's a simple example, that helps many.

But I'm still trying to figure out why this syntax is fine in `MetricsFilter.java` and not with the servlet. I added it for consistency.

Reply:
Hi @brian-brazil,

I'll see how to rephrase this.

But it is supposed to work on any web app container that supports the servlet 3.0 specs (which is referred by the pom.xml of this module). 

Cheers